scripts/install/run.sh

```diff
@@ -0,0 +1,343 @@
+    ( cd ${CLONE_DIR} && yarn build:core:lib)
+
+    printh "Updating projects versions."
+    update_projects_versions ${SPARTACUS_PROJECTS[@]}
+
+    verdaccio --config ./config.yaml &
+
+    VERDACCIO_PID=$!
+    echo "verdaccio PID: ${VERDACCIO_PID}"
+
+    sleep 45
+
+    printh "Creating core npm package"
+    ( cd ${CLONE_DIR}/dist/core && yarn publish --new-version=${SPARTACUS_VERSION} --registry=http://localhost:4873/ --no-git-tag-version )
+
+    printh "Creating storefrontlib npm package"
+    ( cd ${CLONE_DIR}/dist/storefrontlib && yarn publish --new-version=${SPARTACUS_VERSION} --registry=http://localhost:4873/ --no-git-tag-version )
+
+    printh "Creating storefrontstyles npm package"
+    ( cd ${CLONE_DIR}/projects/storefrontstyles && yarn publish --new-version=${SPARTACUS_VERSION} --registry=http://localhost:4873/ --no-git-tag-version )
+
+    printh "Creating assets npm package"
+    ( cd ${CLONE_DIR}/dist/assets && yarn publish --new-version=${SPARTACUS_VERSION} --registry=http://localhost:4873/ --no-git-tag-version )
+
+    printh "Creating schematics npm package"
+    ( cd ${CLONE_DIR}/projects/schematics && yarn && yarn build && yarn publish --new-version=${SPARTACUS_VERSION} --registry=http://localhost:4873/ --no-git-tag-version )
+
+    create_apps
+
+    sleep 5
+
+    (kill ${VERDACCIO_PID} || echo "Verdaccio not running on PID ${VERDACCIO_PID}. Was it already runnig before starting the script?")
+
+    npm set @spartacus:registry https://registry.npmjs.org/
+    echo "Finished: npm @spartacus:registry set back to https://registry.npmjs.org/"
+}
+
+function prestart_csr {
+    if [ -z "$CSR_PORT" ]; then
+        echo "Skipping prestart csr script"
+    else
+        printh "Prestart setup for csr app"
+        ( cd ${INSTALLATION_DIR}/csr && yarn build )
+    fi
+}
+
+function prestart_ssr {
+    if [ -z "$SSR_PORT" ]; then
+        echo "Skipping prestart ssr script"
+    else
+        printh "Prestart setup for ssr app"
+        ( cd ${INSTALLATION_DIR}/ssr && yarn build && yarn build:ssr )
+    fi
+}
+
+function prestart_ssr_pwa {
+    if [ -z "$SSR_PWA_PORT" ]; then
+        echo "Skipping prestart ssr script (with pwa support)"
+    else
+        printh "Prestart setup for ssr app (with pwa support)"
+        ( cd ${INSTALLATION_DIR}/ssr_pwa && yarn build && yarn build:ssr )
+    fi
+}
+
+function start_csr_unix {
+    if [ -z "$CSR_PORT" ]; then
+        echo "Skipping csr app start"
+    else
+        prestart_csr
+        printh "Starting csr app"
+        pm2 start --name "${BASE_SITE}-csr-${CSR_PORT}" serve -- ${INSTALLATION_DIR}/csr/dist/csr/ --single -p ${CSR_PORT}
+    fi
+}
+
+function start_ssr_unix {
+     if [ -z "$SSR_PORT" ]; then
+        echo "Skipping ssr app start"
+    else
+        prestart_ssr
+        printh "Starting ssr app"
+        ( cd ${INSTALLATION_DIR}/ssr && export PORT=${SSR_PORT} && pm2 start --name "${BASE_SITE}-ssr-${SSR_PORT}" dist/ssr/server/main.js )
```

Comment:
This SSR works without the NODE_TLS.. flag?

Reply:
I guess it shouldn't. Will add it here as well.